### PR TITLE
Feat/add copy clipboard

### PR DESCRIPTION
**Pull request by maya-fisher**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [unreleased]
 - hotfix: ([148](https://github.com/meateam/drive-ui/pull/148)): fix page 503 bug
 
+- minor: ([145](https://github.com/meateam/drive-ui/pull/145)): add copy to clipboard.
+
```
Comment (yoavrozner) [inline CHANGELOG.md:12]:
```suggestion
```
One is enough 😆

File: src/i18n/en.json [hunk 2]
```diff
@@ -96,7 +96,8 @@
     "MoreInfo": "for more info, press here",
     "BackToShare": "return to select approvers",
     "Favorite": "add favorite",
-    "RemoveFavorite": "remove favorite"
+    "RemoveFavorite": "remove favorite",
+    "CopyToClipboard": "copy to clipboard"
```
Comment (yoavrozner) [inline src/i18n/en.json:100]: This is missing in the Hebrew

**Review by yoavrozner — CHANGES_REQUESTED** [review]
Good job! Please see my comments

File: src/i18n/en.json (hunk 2 above)
Comment (maya-fisher) [reply src/i18n/en.json:100]: Done.

**Review by yoavrozner — APPROVED** [review]
Good job!

**Pull request merged**